db/helpers/common.js

```diff
@@ -265,10 +265,9 @@ function setIsDeleted(Promise, inst) {
  */
 function validateObject(object, schema) {
   const options = { additionalProperties: false };
```

Comment:
Looks like options is an unused variable.

Reply:
Yeah I missed that. NIce catch